modules/pensions/spec/lib/benefits_intake/pension_benefit_intake_job_spec.rb

```diff
@@ -60,8 +72,9 @@
         expect(job).to receive(:send_confirmation_email)
         expect(job).not_to receive(:send_submitted_email)
         expect(job).to receive(:cleanup_file_paths)
-
-        job.perform(claim.id, :user_uuid)
+        expect do
+          job.perform(claim.id, :user_uuid)
+        end.to change(Kafka::EventBusSubmissionJob.jobs, :size).by(1)
```

Comment:
@madebydna I found that the `perform_async` method does not check for the correct parameters so that's why the specs were passing because it was only checking if `perform_async` was called, not if it was called with the correct parameters. Even after correcting the call, This spec is still failing and I'm not able to see the job queue get added to. Do you see what I'm doing wrong?

Reply:
Thank you! That was driving me crazy.